contracts/minter/src/contract.rs

```diff
@@ -442,14 +419,18 @@ pub fn execute_update_start_time(
         ));
     }
 
-    let default_start_time = Expiration::AtTime(Timestamp::from_nanos(GENESIS_MINT_START_TIME));
-    let start_time = if start_time < default_start_time {
-        default_start_time
+    if config.start_time.is_expired(&env.block) {
+        return Err(ContractError::AlreadyStarted {});
+    }
+
+    let genesis_start_time = Expiration::AtTime(Timestamp::from_nanos(GENESIS_MINT_START_TIME));
```

Comment:
currently whitelist does not have genesis start time validation, maybe add that too or open separate issue

Reply:
@yubrew Do you mind creating a new issue for this? This PR is getting quite long.